exercise-integration-testing-5-53/test/spread-the-code-for-task-10/when-event-is-valid.test.js

```diff
@@ -0,0 +1,270 @@
+// const request = require("supertest");
+const axios = require("axios");
+const { initializeAPI } = require("../../sensors-api");
+const nock = require("nock");
+
+let App;
+let port;
+let server;
+
+beforeAll(async () => {
+  const { expressApp, thePort, theServer } = await initializeAPI();
+  App = expressApp;
+  port = thePort;
+  server = theServer;
+});
+
+afterAll((done) => {
+  server.close(() => {
+    done();
+  });
+});
+
+beforeEach(() => {
+    nock("http://localhost").get("/notification").reply(200, {
+      success: true,
+    });
+  });
+  
+  function helper(length) {
```

Comment:
Great function. Can be even more compact with just `${math.random()*10000}{process.id}'